study.md

```diff
@@ -85,7 +97,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
+"My name is #{dale[:game_name]} #{dale[:family_name]} and I'm an #{dale[:occupation]} that enjoys #{dale[:hobbiles[0]]} and #{dale[:hobbiles[1]]}"
```

Comment:
`dale[:game_name]` should be `dale[:given_name]`.  Since you're trying to access elements of an array, the index needs to come after the array that's being returned from accessing the hash's key's value.  `dale[:hobbiles[0]]` should be `dale[:hobbiles][0]`.